src/tests/base/test_orders.py

```diff
@@ -543,6 +543,13 @@ def test_not_sent_after_reminder_date(self):
         send_download_reminders(sender=self.event)
         assert len(djmail.outbox) == 0
 
+    @classscope(attr='o')
+    def test_not_sent_for_disabled_sales_channel(self):
+        self.event.settings.mail_days_download_reminder = 2
+        self.event.settings.mail_sales_channel_ticket_reminder = []
+        send_download_reminders(sender=self.event)
+        assert len(djmail.outbox) == 0
+
```

Comment:
So I thought this test would fit here. I did not find a nice place to test the `placed_paid` option.